src/org/gh/RunMe.java

```diff
@@ -12,28 +12,34 @@ public static void main(String args[]){
         System.out.println("Creating a new student and Teacher");
         Student objStudent = new Student("Student 1", "GradeA");
```

Comment:
If you rename "Student" to Stud, you'll need to update this (Same with next line for "Teacher" vs "Teach"